docs/application/web/guides/connectivity/bluetooth.md

```diff
@@ -697,8 +757,531 @@ To read and write a value of the Bluetooth descriptor:
    descriptor.writeValue(newValue, writeSuccess, writeFail);
    ```
 
+### Accessing the ATT MTU of connected device
+
+To get the ATT MTU value or request change of the ATT MTU value:
+
+1. [Connect to a Bluetooth Low Energy device](#connecting-to-a-bluetooth-low-energy-device).
+2. If the `device` has been connected call `getAttMtu()` function on the `device` object to get current ATT MTU value:
+   ```
+   var attMtu = device.getAttMtu();
+   ```
+3. If the `device` has been connected call `requestAttMtuChange()` function on the `device` object to request a change of the ATT MTU value. Pass desired ATT MTU value as an argument:
```

Comment:
3. If the `device` is connected, call `requestAttMtuChange()` on the `device` object to request a change of the ATT MTU value. Pass desired ATT MTU value as an argument:

Reply:
I have applied your suggestions.